rubocop_test.rb

```diff
@@ -0,0 +1,35 @@
+# This class is a test of Rubocop linting
+#
+class RubocopTest
+  # GOOD: trailing dots
+  'Hello world'.
+    sub('world', 'evenyone')
+  # BAD: leading dots
+  'Hello world'
+    .sub('world', 'evenyone')
+
+  # GOOD: new style hash syntax, with no mixed keys
+  _good_hash = { foo: 'foo' }
+  # BAD: old style hash syntax
+  _bad1_hash = { :bar => 'bar' }
+  # BAD: mixed keys syntax
+  _bad2_hash = { foo: 'foo', :bar => 'bar' }
+
+  # GOOD: lines over 80 characters
+  _short_line = 'This line is under 80 characters.'
+  # BAD: lines over 80 characters
+  _long_line = 'This line is going to really long and be over the 80 characters.'
+
+  # GOOD: raise only signal exception
+  begin
+    raise
+  rescue StandardError
+    # handle it
+  end
+  # BAD: fail only signal exception
+  begin
+    fail
+  rescue StandardError
```

Comment:
Hadn't intended this to be checked - not picked up locally when running `bundle exec rubocop -D rubocop_test.rb`

Reply:
This wasn't raised in the GitHub action runner, I think because Hound is using Rubocop 0.80.1, instead of 0.81.0 - there was this change https://github.com/rubocop-hq/rubocop/pull/7805

Doesn't explain the other consistencies tho.